packages/flutter/lib/src/material/tabs.dart

```diff
@@ -49,6 +49,40 @@ enum TabBarIndicatorSize {
   label,
 }
 
+/// Defines how tabs are aligned horizontally in a [TabBar].
+///
+/// See also:
+///
+///   * [TabBar], which displays a row of tabs.
+///   * [TabBarView], which displays a widget for the currently selected tab.
+///   * [TabBar.tabAlignment], which defines the horizontal alignment of the
+///     tabs within the [TabBar].
+enum TabAlignment {
+  /// When [TabBar.isScrollable] is true, tabs are aligned to the
+  /// start of the [TabBar] with a horizontal offset of 56.0 pixels.
```

Comment:
Is this 56 pixels defined as a constant somewhere? Reading it, I wondered, why 56?

Reply:
The Web page specifics there should be an offset and I can see the offset when testing Compose's TabRow.

It should be 52 pixels and [the value is extracted from Compose's source code.](https://github.com/androidx/androidx/blob/f506fea2025739cf026364adbe742149e3a4e62b/compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/TabRow.kt#L493-L497)